mysql_scripts/table_creation/FullDB.sql

```diff
@@ -73,9 +73,16 @@ CREATE TABLE Client (
 
 CREATE TABLE Appointment (
 	appointmentId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
-	scheduledTime DATETIME NOT NULL,
+    scheduledTime DATETIME NOT NULL,
 	createdAt DATETIME NOT NULL DEFAULT NOW(),
-	arrivedAt DATETIME NULL,
+    language VARCHAR(255) NOT NULL,
+	timeIn DATETIME NULL DEFAULT NULL,
+    timeReturnedPapers DATETIME NULL DEFAULT NULL,
+    actualAppointmentTime DATETIME NULL DEFAULT NULL,
+    timeFinished DATETIME NULL DEFAULT NULL,
+    completed BOOLEAN NOT NULL DEFAULT FALSE,
+    notCompleted BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
Hm I'm not sure how I feel about these `completed` and `notCompleted` columns. Because if completed is true, then notCompleted is obviously false and vice versa. The only edge case I see is that case where it's neither completed nor notCompleted (as in they haven't had the appointment yet), and I think NULL is a totally fine way to represent that situation. 
It's normally bad practice in a database to have fields that more or less depend on each other (like if we had a firstName and a lastName column, and then also a fullName column--it's redundant and kinda bad because then every time you update one field, you need to remember to change the other).
So I would be partial to removing either notCompleted or completed.